### DISCUSSION TITLE
Warning: Not farming any plots on this harvester. Check your configuration

### DISCUSSION THREAD

**Discussion by fidibert** [Chia-Network/chia-blockchain#3484] [Support]
I use 2 different Networks with each 1 Plotter/Harvester with the same Private Key. Now i got this Warning in just one of the two machines. "Not farming any plots on this harvester. Check your configuration.

Can anyone help me with this problem?

Cheers

**Comment by renedavalos** [comment]
I have the same problem.. did you fix it?

**Comment by JordyBaylac** [comment]
same problem

**Comment by miodzicho** [comment]
same

**Comment by Luuuxury** [comment]
if anyone made it ?

**Comment by lixiaodong** [comment]
the same

**Comment by D2BO** [comment]
To fix this please see my post "Fix Missing Plot"

**Comment by D2BO** [comment]
To fix this issue.
Edit config file
Search the word "Null"
replace the null where your wallet supposed to go with you wallet address.
See Below:

pool:
logging: *XXXX
network_overrides: *XXXX
selected_network: mainnet
xch_target_address: Null (Wallet Address goes here)

**Comment by BrandtH22** [comment]
@fidibert , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
